src/RudderClient.ts

```diff
@@ -193,19 +193,26 @@ async function group(groupId: string, traits: Object | null = null) {
     logWarn("group: 'groupId' must be a string");
     return;
   }
-  logWarn("group: Method not supported");
+  bridge.group(groupId, traits, options)
 }
 
-async function alias(previousId: string, userId: string) {
-  if (previousId == undefined) {
-    logWarn("alias: Mandatory field 'previousId' missing");
+async function alias(previousId: string, userId: string): Promise<void>; 
```

Comment:
`alias(previousId, userId)` is not adhering to our spec and also we are not using the value of `previousId` anywhere in the implementation. Can we remove it ?

Reply:
The API was present earlier. We are not sure if anyone is using it or not.! Updating to this version can break their implementation. We should be compatible with the previous version.

Reply:
Got it.